pipelines/LSSTComCamSim/nightly-validation.yaml

```diff
@@ -15,15 +15,8 @@ tasks:
       connections.data: preSourceTable_visit
       connections.inputName: preSourceTable_visit
       connections.outputName: preSourceTableCore
-  calibrate:
-    class: lsst.pipe.tasks.calibrate.CalibrateTask
-    config:
-      python: |
-        from lsst.analysis.tools.tasks import CalexpSummaryAnalysisTask
-        config.createSummaryMetrics.retarget(CalexpSummaryAnalysisTask)
-        from lsst.analysis.tools.atools import CalexpSummaryMetrics
-        config.createSummaryMetrics.atools.calexpSummaryMetrics = CalexpSummaryMetrics
-      doCreateSummaryMetrics: true
```

Comment:
What did you do about getting these metrics out?

Reply:
There's a separate `PipelineTask` (label `calexpSummary`) that reads the calexp summaries and writes the metrics that was already being used instead in more actively-maintained pipelines, and I've switched this one to behave the same way.